internal/controller/suite_test.go

```diff
@@ -140,7 +140,7 @@ func setupAllWithManager(k8sManager manager.Manager) {
 		Client:        k8sManager.GetClient(),
 		Scheme:        k8sManager.GetScheme(),
 		Tracker:       remoteTrackerMock,
-		RequeuePeriod: time.Millisecond,
+		RequeuePeriod: time.Second,
```

Comment:
Just a random change. 
This makes the tests now take ~26seconds in total, from ~24, however the logs are a thousand times less spammy when things fail.